recipes/slugify/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "slugify" %}
+{% set version = "0.0.1" %}
+{% set md5 = "557a7e401ea3c9716d29389c56cf82c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - slugify
+
+about:
+  home: http://github.com/zacharyvoase/slugify
+  license: Public Domain
+  license_family: Other
+  # license_file: UNLICENSE # When added to MANIFEST.in – https://github.com/zacharyvoase/slugify/pull/4
```

Comment:
Technically we don't have to, but I agree that we should. Thanks for putting that PR together. 👍